example/App.tsx

```diff
@@ -1,31 +1,12 @@
 import 'react-native-gesture-handler'
-import React, { useEffect } from 'react'
-import messaging from '@react-native-firebase/messaging'
+import React from 'react'
 import { AppNavigator } from './src/navigation'
-import { useNotifications } from 'react-native-notificated'
 import { NavigationContainer } from '@react-navigation/native'
 
-const App = () => {
-  const { notify } = useNotifications()
-
-  useEffect(() => {
-    const unsubscribe = messaging().onMessage(async (remoteMessage) => {
-      notify('info', {
-        params: {
-          title: remoteMessage.notification?.title || '',
-          description: remoteMessage.notification?.body,
-        },
-      })
-    })
-
-    return unsubscribe
-  }, [notify])
-
-  return (
-    <NavigationContainer>
-      <AppNavigator />
```

Comment:
lets make sure we appropriately update docs (I can't recall if we have some references there to the code here)

Reply:
We have the same App.tsx code pasted in the docs but without any references.

https://github.com/TheWidlarzGroup/react-native-notificated/pull/161/files#diff-91c4174496e5372d36d2fea6eccf7c9c08b93f624dfad2152cc4f1e79c43bd7f

Reply:
👍🏽